go.mod

```diff
@@ -1,3 +1,11 @@
 module ryepup/advent2021
 
 go 1.17
+
+require (
+	github.com/davecgh/go-spew v1.1.0 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	github.com/stretchr/objx v0.1.0 // indirect
+	github.com/stretchr/testify v1.7.0 // indirect
```

Comment:
is there any equivalent to npm's `devDependencies`?

Reply:
doesn't seem like it, but there's [module graph pruning](https://go.dev/ref/mod#graph-pruning) so consumers of a library won't need to download test modules unless they run call code that depends on it.